README.md

```diff
@@ -45,6 +71,8 @@ const limitd = new Limitd({
 - `buckets` (object): Setup your bucket types.
 - `prefix` (string): Prefix keys in Redis.
 - `ping` (object): Configure ping to Redis DB.
+- `username` (string): Redis username. Only to be used in Cluster mode. Needs Redis >= v6.
```

Comment:
if not in cluster mode, this is just silently ignored, right?

Do we want to error? If not, maybe we should change this message to:
```suggestion
- `username` (string): Redis username. This is ignored if not in Cluster mode. Needs Redis >= v6.
```

Reply:
done